### ISSUE TITLE
Acute hepatic steatosis of pregnancy

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5881]
**Preferred term label:** An acute form of hepatic steatosis during pregnancy.

**Synonyms** Acute Fatty Liver of Pregnancy, AFLP

**Definition (free text, please give PubMed ID)** An acute form of hepatic steatosis during pregnancy., PMID: 30707120

**Parent term (use hpo.jax.org/app)** 

Acute hepatic steatosis HP:0006573

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is a disease and not a phenotypic feature. There is a MONDO term for the disease
https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0016573
